core/src/main/java/org/axonframework/queryhandling/QueryBus.java

```diff
@@ -104,17 +89,68 @@
 
     /**
      * Dispatch the given {@code query} to a single QueryHandler subscribed to the given {@code query}'s
-     * queryName/initialResponseType/updateResponseType.
+     * queryName/initialResponseType/updateResponseType. The result is lazily created and until there is a subscription
+     * to initial result there will be no execution of query handler. In order not to miss updates, query bus will
+     * queue all updates which happen after subscription query is done and once the subscription to the flux is made,
+     * these updates will be emitted.
+     * <p>
+     * Backpressure mechanism to be used is {@link SubscriptionQueryBackpressure#defaultBackpressure()}. The size of
+     * buffer which accumulates the updates (not to be missed) is {@link Queues#SMALL_BUFFER_SIZE}.
+     * </p>
+     *
+     * @param query the query
+     * @param <Q>   the payload type of the query
+     * @param <I>   the response type of the query
+     * @param <U>   the incremental response types of the query
+     * @return query result containing initial result and incremental updates
+     */
+    default <Q, I, U> SubscriptionQueryResult<QueryResponseMessage<I>, SubscriptionQueryUpdateMessage<U>> subscriptionQuery(
+            SubscriptionQueryMessage<Q, I, U> query) {
+        return subscriptionQuery(query, SubscriptionQueryBackpressure.defaultBackpressure(), Queues.SMALL_BUFFER_SIZE);
+    }
+
+    /**
+     * Dispatch the given {@code query} to a single QueryHandler subscribed to the given {@code query}'s
```

Comment:
The same javadoc comments as in the function above would hold for this bit of javadoc.